docs/adv-tool-usage.html

```diff
@@ -656,5 +659,27 @@ <h3>Go compilers and Advance Toolchain</h3>
     <div class="ibm-col-2-2 ibm-center ibm-padding-top-r1">
 <p class"ibm-center"><button type="button" class="ibm-btn-pri ibm-btn-gray-50 copyURL12">Copy Link</button></p></div>
 </div>
+</div>
+
+ <!-- Begin content for overview tab -->
+
+<div id="faq13" class="ibm-tabs-content">
+<h3>Environment modules</h3>
+
+<p>Environment Modules is a tool that simplify shell initialization and lets users easily modify their environment during the session with modulefiles.</p>
+
+<p>Since AT 11.0, modulefiles are provided in the devel and cross-common packages to help set easily the environment for use AT.</p>
+
+<p>To use it simple run the command:</p>
+<pre>    module load atX.X </pre>
+
+<p>To stop using it is also simple:</p>
```

Comment:
s/To stop using it is also simple/To stop using, it is also simple/ # just added a comma